mount.go

```diff
@@ -99,6 +101,25 @@ be interacting with the file system.`)
 	}
 	bm := gcsx.NewBucketManager(bucketCfg, storageHandle)
 
+	if mountConfig.StatCacheMaxSizeInMb != config.StatCacheMaxSizeInMbUnsetSentinel {
+		if mountConfig.StatCacheMaxSizeInMb < -1 {
+			return nil, fmt.Errorf(config.StatCacheMaxSizeInMbInvalidValueError)
+		}
+
+		if mountConfig.StatCacheMaxSizeInMb == -1 {
+			bucketCfg.StatCacheMaxSizeMiB = math.MaxUint64
```

Comment:
Just wondering max supported StatCacheMaxSizeInMb = math.MaxUint64 >> 20 right?

Reply:
it was a bug. fixed it.